src/code/z_kankyo.c

```diff
@@ -2103,9 +2103,9 @@ void Environment_PlaySceneSequence(PlayState* play) {
     }
 
     PRINTF("\n-----------------\n", ((void)0, gSaveContext.forcedSeqId));
-    PRINTF("\n 強制ＢＧＭ=[%d]", ((void)0, gSaveContext.forcedSeqId)); // "Forced BGM"
+    PRINTF(T("\n 強制ＢＧＭ=[%d]", "\n Forced BGM=[%d]"), ((void)0, gSaveContext.forcedSeqId));
     PRINTF("\n     ＢＧＭ=[%d]", play->sceneSequences.seqId);
```

Comment:
missed this guy
```suggestion
    PRINTF(T("\n     ＢＧＭ=[%d]", "\n     BGM=[%d]"), play->sceneSequences.seqId);
```

Reply:
I skip these fullwidth (https://en.wikipedia.org/wiki/Halfwidth_and_fullwidth_forms) latin only strings but there's an argument to also "translate" them to ascii yeah
I'd rather do a separate PR for all of them for clarity and motivating it